docs/user_guide/datasets/dataset_zoo.md

```diff
@@ -88,8 +88,10 @@ train_stats, _, _ = model.train(training_set=training_set, test_set=test_set, mo
 Here is the list of the currently available datasets:
 
 | Dataset                                   | Hosted On             | Description                                                                                      |
-|-------------------------------------------|-----------------------|--------------------------------------------------------------------------------------------------|
+| ----------------------------------------- | --------------------- | ------------------------------------------------------------------------------------------------ |
 | adult_census_income                       | archive.ics.uci.edu   | <https://archive.ics.uci.edu/ml/datasets/adult>. Whether a person makes over $50K a year or not. |
+| allstate_claims_severity                  | Kaggle                | <https://www.kaggle.com/c/allstate-claims-severity>                                              |
+| amazon_employee_access_challenge          | Kaggle                | <https://www.kaggle.com/c/amazon-employee-access-challenge>                                      |
```

Comment:
Do we know if these are the only new ones that have landed in 0.6? I know @dantreiman has been adding a lot of new datasets to Ludwig so just wanted to make sure. Can deal with this in a follow up PR, not a blocker in any way.

Reply:
Looks like all of the datasets from https://github.com/ludwig-ai/ludwig-docs/pull/210/files are present.